bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/connection/HueBridge.java

```diff
@@ -67,25 +98,31 @@
 
     private static final String DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ss";
 
+    private final HttpClient httpClient;
     private final String ip;
     private final String baseUrl;
     private @Nullable String username;
+    private int timeout = 1000;
```

Comment:
Looks like a strange default value.
Is it a value in seconds ?

Reply:
Yes, we are talking about seconds. I will decrease it to 10.

Reply:
But it is then used as seconds in the code if I correctly remember when I reviewed.

Reply:
Yes, i changed that as well. See https://github.com/openhab/openhab-addons/pull/11842/commits/0ad986ad41599d61bbf707b11b12e15e3d828ba6.